### PR TITLE
Bowl ritual recipes

### PR DESCRIPTION
**Pull request by reicheltp**
Load recipes for bowl rituals from JSON resources. 

Currently, only supports items to items rituals.

### CODE REVIEW

File: src/main/resources/data/celtic_rituals/recipes/healing_ritual.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+      "item": "minecraft:redstone"
+    },
+    {
+      "item": "minecraft:gold_nugget"
+    }
+  ],
+  "result": {
+    "item": "minecraft:glowstone_dust"
+  }
+}
```
Comment (Sebkasanzew) [inline src/main/resources/data/celtic_rituals/recipes/healing_ritual.json]: Please an empty line at the end

**Comment by reicheltp** [comment]
@raphaelzoellner @Sebkasanzew Should we merge this as minimal PoC or wait for #19 to be done?

File: src/main/kotlin/com/reicheltp/celtic_rituals/rituals/bowl/RitualBowlBlock.kt [hunk 2]
```diff
@@ -132,9 +136,11 @@ class RitualBowlBlock : Block(
                 player.heldItemMainhand.damageItem(1, player, { it.sendBreakAnimation(handIn) })
             }
 
+            val recipe = worldIn.server!!.recipeManager.getRecipe<IInventory, BowlRitualRecipe>(ModRecipes.BOWL_RITUAL_TYPE!!, tile, worldIn)
+
             if (!inProgress) {
                 worldIn.setBlockState(pos, state.with(BlockStateProperties.ENABLED, true))
-                worldIn.pendingBlockTicks.scheduleTick(pos, ModBlocks.RITUAL_BOWL!!, 200)
+                worldIn.pendingBlockTicks.scheduleTick(pos, ModBlocks.RITUAL_BOWL!!, recipe.map { it.duration }.orElse(BowlRitualRecipe.DEFAULT_DURATION))
```
Comment (Sebkasanzew) [inline src/main/kotlin/com/reicheltp/celtic_rituals/rituals/bowl/RitualBowlBlock.kt]: Maybe add more line breaks to increase readability

**Comment by Sebkasanzew** [comment]
> @raphaelzoellner @Sebkasanzew Should we merge this as minimal PoC or wait for #19 to be done?

I think, at this early stage of development, we can also merge small changes and clean up later as soon as we see some working patterns evolve.

File: src/main/kotlin/com/reicheltp/celtic_rituals/rituals/bowl/RitualBowlBlock.kt (hunk 2 above)
Comment (reicheltp) [reply src/main/kotlin/com/reicheltp/celtic_rituals/rituals/bowl/RitualBowlBlock.kt]: Formatting is enforced by ktlint. If there are any additional rules, we should extend the ruleset.

File: src/main/resources/data/celtic_rituals/recipes/healing_ritual.json (hunk 1 above)
Comment (reicheltp) [reply src/main/resources/data/celtic_rituals/recipes/healing_ritual.json]: This should also be done by a linter. See #21

File: src/main/kotlin/com/reicheltp/celtic_rituals/rituals/bowl/RitualBowlBlock.kt (hunk 2 above)
Comment (reicheltp) [reply src/main/kotlin/com/reicheltp/celtic_rituals/rituals/bowl/RitualBowlBlock.kt]: Added #21 to keep an eye on this.

**Comment by reicheltp** [comment]
Also fixes #22 

Relates to #19

**Comment by reicheltp** [comment]
image

**Pull request merged**